internal/server/statvar/fetcher/formula_fetcher.go

```diff
@@ -39,83 +41,108 @@ func FetchFormulas(
 	remoteRespChan := make(chan *pbv2.NodeResponse, 1)
 	// Fetch for BT and SQL.
 	errGroup.Go(func() error {
-		data, _, err := v1pv.Fetch(
-			errCtx,
-			store,
-			[]string{"StatisticalCalculation"},
-			[]string{"typeOf"},
-			0,
-			"",
-			"in",
-		)
-		if err != nil {
-			return err
-		}
 		statCal := []string{}
-		for _, nodes := range data["StatisticalCalculation"]["typeOf"] {
-			for _, node := range nodes {
-				statCal = append(statCal, node.Dcid)
+		nextToken := ""
+		for ok := true; ok; ok = (nextToken != "") {
```

Comment:
nit: would it be cleaner to do something like:

```
for {
  ...
  if nextToken == "" {
    break
  }
}
```

Reply:
Done